README.md

```diff
@@ -288,9 +288,12 @@ ${hostedzone://example.com/private}
 
 Looks up an AMI given a name prefix which may contain wildcards. If query returns multiple images the latest is used.
 
+Owners takes a query string list of AWS account ID, self (owner is the sender of the request), or an AWS owner alias (valid values are amazon | aws-marketplace | microsoft). Omitting this option returns all images for which you have launch permissions, regardless of ownership.
```

Comment:
Big fan of this, needed the flexibility to resolve images that are owned by another account and shared.